scripts/create-local-tag-for-release

```diff
@@ -94,16 +99,23 @@ sync_local_tags_from_remote() {
     # fetch remote tags
     git fetch the-real-upstream --tags
 
+    # record the latest release tag in remote, before creating a new tag
+    CURR_REMOTE_RELEASE_TAG=$(get_latest_tag)
+
     # clean up tracking
     git remote remove the-real-upstream
 }
 
 create_tag() {
     git tag $NEW_TAG
-    echo -e "\n✅ Created new tag $NEW_TAG (Current latest release tag: v$LATEST_TAG)\n"
+    echo -e "\n✅ Created new tag $NEW_TAG (Current latest release tag in remote: v$CURR_REMOTE_RELEASE_TAG)\n"
```

Comment:
the tags should have the prepended `v` so this would print `vv1.1.0` right?

Reply:
$CURR_REMOTE_RELEASE_TAG doesnt include the 'v', comes from `make -s -f $MAKEFILE_PATH latest-release-tag | cut -b 2-` -- removing the 'v' because this script is interested in the numbers only. 

Thats why you will see all prints / final $NEW_TAG assignments include 'v'. 


Do you think a comment around `get_latest_tag` is required?

Reply:
I'm good with it since the output is correct :+1: